intro-to-tidyverse.rmd

```diff
@@ -930,9 +946,11 @@ The goal of tidyr is to help you create tidy data. Tidy data is data where:
 --
 
 
-### Make Taller and Make Wider
-* `gather()` - takes multiple columns, and gathers them into key-value pairs: it makes “wide” data longer.
-* `spread()` - takes two columns (key & value), and spreads into multiple columns: it makes “long” data wider.
+### Make Longer and Make Wider
+* `pivot_longer()` - takes multiple columns, and makes the data "longer" by reducing the number of columns and increasing the number of rows.
```

Comment:
did my best to take from `vignettes("pivot")`. The functions are much more flexible now and as such previous definitions are too restrictive.